apps/frontend/src/app/core/navbar/navbar.component.html

```diff
@@ -1,44 +1,98 @@
+                {{ theme.name }}
+                <i [class.bi-check]="(theme$ | async) === theme.value"></i>
+              </button>
+            </div>
+          </div>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href="https://github.com/Morphclue/apollusia" target="_blank">
-            <i class="bi-github icon" title="GitHub"></i>
-          </a>
+          <a class="nav-link bi-github" href="https://github.com/Morphclue/apollusia" target="_blank" ngbTooltip="GitHub"></a>
         </li>
       </ul>
     </div>
   </div>
 </nav>
+<ng-template #content let-offcanvas>
+  <div class="offcanvas-header">
+    <h4 class="offcanvas-title" id="offcanvas-basic-title">
+      <img class="navbar-logo" ngSrc="/assets/logo.png" alt="Apollusia Logo" width="32" height="32">
+      Apollusia
+    </h4>
+    <button type="button" class="btn-close" aria-label="Close" (click)="offcanvas.dismiss()"></button>
+  </div>
+  <div class="offcanvas-body">
+    <ul class="nav nav-pills flex-column h-100">
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-house" routerLinkActive="active" routerLink="dashboard">
+          Dashboard
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-calendar-plus" routerLinkActive="active" routerLink="poll/create">
+          Create Poll
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-gear" routerLinkActive="active" routerLink="settings">
+          Settings
+        </a>
+      </li>
+      <hr/>
+      <h6>
+        Recent Polls
+      </h6>
+      @for (poll of recentPolls; track poll.id) {
+        <li class="nav-item w-100">
+          <a class="nav-link text-truncate" routerLinkActive="active" [routerLink]="['poll', poll.id, 'participate']">
+            <i [class]="poll.location | locationIcon"></i>
+            {{ poll.title }}
+            <app-location-link class="small text-muted" [location]="poll.location" [shorten]="true"></app-location-link>
+          </a>
+        </li>
+      }
+      <hr class="mt-auto"/>
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-info-circle" routerLinkActive="active" routerLink="about">
+          About
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-file-earmark-text" routerLinkActive="active" routerLink="legal">
+          Legal
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-github" href="https://github.com/Morphclue/apollusia">
+          GitHub
+        </a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-item nav-link bi-bug" href="https://github.com/Morphclue/apollusia/issues/new?template=bug_report.md" target="_blank">
+          Report a bug
+        </a>
+      </li>
+      <div class="small text-muted text-center">
+        Copyright © Apollusia, 2024
```

Comment:
Wouldn't it be better to implement the current year here instead of a hardcoded value that needs to be adjusted every year? (looking at you fulib 👀)

Reply:
Yes, I will see if there is a good solution that does not just calculate the year on the user's time. I'd like it to be at build time